src/uct/tcp/tcp_iface.c

```diff
@@ -155,6 +155,23 @@ unsigned uct_tcp_iface_progress(uct_iface_h tl_iface)
     return count;
 }
 
+static ucs_status_t uct_tcp_iface_flush(uct_iface_h tl_iface, unsigned flags,
+                                        uct_completion_t *comp)
+{
+    uct_tcp_iface_t *iface = ucs_derived_of(tl_iface, uct_tcp_iface_t);
+
+    if (comp != NULL) {
+        return UCS_ERR_UNSUPPORTED;
+    }
+
+    if (iface->outstanding) {
+        UCT_TL_IFACE_STAT_FLUSH_WAIT(&iface->super);
```

Comment:
looks like need to return `UCS_INPROGRESS` or `UCS_ERR_NO_RESOURCE` here.  Should not it be caught by UCT flush test?

Reply:
i introduced this bug when tried to fix coverity :( gtest catch it, but after a very large amount of iterations..